ads/index.js

```diff
@@ -87,6 +91,24 @@ function onAdsComplete (event) {
 	}
 }
 
+function sendAdLoadedTrackingEvent() {
+	const marks = performance.getEntriesByType ?
+		performance.getEntriesByType('mark')
```

Comment:
I think you can also do `getEntriesByName` here, and saving having to look through the array